hack/ci/e2e.sh

```diff
@@ -21,11 +21,12 @@
 set -o errexit
 set -o nounset
 set -o pipefail
+set -o xtrace
 
 # our exit handler (trap)
 cleanup() {
     # always attempt to dump logs
-    kind export logs "${ARTIFACTS}/logs" || true
+    kind "export" logs "${ARTIFACTS}/logs" || true
```

Comment:
yikes, why is it doing that...

Reply:
because "export" is a keyword in bash. possibly we could rename it... :thinking:

Reply:
but this works fine for me:
```
#!/usr/bin/env bash

set -o errexit
set -o nounset
set -o pipefail
set -o xtrace

cleanup() {
  kind export logs
}
cleanup
```

Reply:
hmm you're right, this fix didn't work, it must have just been a coincidence.

.. not sure if I should spend more time on this or kubetest2

Reply:
http://gcsweb.k8s.io/gcs/kubernetes-jenkins/logs/ci-kubernetes-kind-conformance-parallel/9528/artifacts/ <-- no logs